v03_pipeline/lib/tasks/write_project_family_tables.py

```diff
@@ -26,27 +27,26 @@ def complete(self) -> bool:
             for write_family_table_task in self.dynamic_write_family_table_tasks
         )
 
-    def run(self):
-        # https://luigi.readthedocs.io/en/stable/tasks.html#dynamic-dependencies
-        # Fetch family guids from project table
-        update_project_table_task: luigi.Target = yield self.clone(
-            UpdateProjectTableTask,
-        )
-        project_ht = hl.read_table(update_project_table_task.path)
-        family_guids_in_project_table = set(hl.eval(project_ht.globals.family_guids))
+    def requires(self) -> list[luigi.Task]:
+        return [
+            self.clone(
+                WriteRemappedAndSubsettedCallsetTask,
+            ),
+            self.clone(
+                UpdateProjectTableTask,
+            ),
+        ]
 
-        # Fetch family guids from pedigree
```

Comment:
The specific failure was a family existing in the pedigree, failing a check and thus being excluded from the rest of the pipeline, but being present on the  project table from a different callset.